JS/cart.js

```diff
@@ -1,6 +1,19 @@
-let nb_article_by_createCart = 0;
+/*
+ * total_price : variable used to know the final price to pay for the cart.
+ * It's a global var because it will be used when an article is created
+ * and also when the quantity of the article is updated.
```

Comment:
global variable need an explanation about why they **can't** be local :)

Reply:
cart_div is not a global variable anymore
Explanation added.